conf/messages.en

```diff
@@ -138,6 +138,43 @@ trustHaveAUTR.heading = Does the trust have a Unique Taxpayer Reference (UTR)?
 trustHaveAUTR.checkYourAnswersLabel = Does the trust have a Unique Taxpayer Reference?
 trustHaveAUTR.error.required = Select yes if the trust has a Unique Taxpayer Reference
 
+whichIdentifier.title = Which unique identifier does the trust have?
+whichIdentifier.heading = Which unique identifier does the trust have?
+whichIdentifier.error.required = Select if the trust has a unique identifier
+whichIdentifier.utr = Unique Taxpayer Reference (UTR)
+whichIdentifier.utr.hint = This is 10 numbers, for example 12345 67890.
+whichIdentifier.urn = Unique Reference Number (URN)
+whichIdentifier.urn.hint = This is 15 numbers and characters, for example ABTRUST12345678.
```

Comment:
Me being pedantic, but possibly a question for design: characters (to me at least) includes numbers. Could this say 'alphanumeric characters' or 'numbers and letters' ?